examples/lighting-and-environment.html

```diff
@@ -133,6 +132,148 @@ <h2>When both <span class="attribute">background-image</span> and <span class="a
     </div>
   </div>
 
+
+  <div class="sample">
+    <div class="demo" id="demo-container-5"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="index">5</div>
+        <h2>Use <span class="attribute">environment-image</span> to set an
+        environtal reflection without changing the background</h2>
```

Comment:
s/environtal/environmental

Reply:
Possibly 'to set environmental lighting without changing the background' (but not sure if lighting or reflection is used throughout)

Reply:
Will fix